openfisca_core/taxbenefitsystems.py

```diff
@@ -48,10 +55,10 @@ class VariableNameConflict(Exception):
 
 class TaxBenefitSystem(object):
     """
-    Represents the legislation.
+    Represents the legislation. It stores parameters (values defined for everyone) and variables (values defined for some given entity e.g. a person).
```

Comment:
J'enlèverait "Represents the legislation" --> cela reste déroutant, quand legislation peut représenter  seulement les paramètres dans certain contextes